.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,7 @@
+Fixes #{Issue Number}
```

Comment:
@AkshGautam Why are you adding curly brackets ({}) here?

Reply:
@luckyman20 this format is taken from mifos mobile template.Should I change?